abdulfatai360/banka
# #165483451 API for getting an account transaction history

#### What does this PR do?
Have an endpoint for getting transaction history of an account

#### Description of Task to be completed?
- Write API to have the endpoint working
- Write test cases for the endpoint
- Have all test cases pass

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/46/head:ft-view-transaction-history-endpoint-165483451`
- Now that you have a copy of the branch, switch to it using  `git checkout ft-view-transaction-history-endpoint-165483451`
- Your directory will now be an exact copy of the PR
- Run `npm install` to install all project's dependencies onto your local machine
- Type `npm run build` or `npm run build;dev` to run the build script and populate the `dist/` directory containing transpiled code
- Run `npm start` or `npm run start:dev` command to start the development server

#### What are the relevant pivotal tracker stories?
[#165483451](https://www.pivotaltracker.com/story/show/165483451)
